### ISSUE TITLE
Villager Tab: No Texture

### ISSUE THREAD

**Issue by AlexDerProGamer** [way2muchnoise/JustEnoughResources#418]
<img width="198" alt="image" src="https://github.com/way2muchnoise/JustEnoughResources/assets/111710468/4f2d6bdb-89f5-44e3-8063-f483f59a411f">

**Forge Version:**
47.1.0

**Here's my mod list:**
- TextruesRubidiumOptions-1.0.3+mc1.20.1
- xaeroarrowfix-forge-1.2+1.20
- Xaeros_Minimap_23.7.0_Forge_1.20
- XaerosWorldMap_1.34.0_Forge_1.20
- appleskin-forge-mc1.20.1-2.5.0
- caelus-forge-3.1.0+1.20
- canary-mc1.20.1-0.2.7
- chunkloaders-1.2.7-forge-mc1.20
- Chunky-1.3.92
- Clumps-forge-1.20.1-12.0.0.3
- collective-1.20.1-6.65
- ContainerSearch-1.0.0-Forge-1.20
- Controlling-forge-1.20.1-12.0.2
- corpse-1.20.1-1.0.5
- create_enchantment_industry-1.20.1-for-create-0.5.1.d-1.2.6.b
- create_misc_and_things_ 1.20.1_4.0A
- create-1.20.1-0.5.1.d
- createaddition-1.20.1-1.0.0b
- create-confectionery1.20.1_v1.0.9
- createendertransmission-2.0.4-1.20.1
- create-stuff-additions1.20.1_v2.0.4a
- cupboard-1.20.1-1.9
- curios-forge-5.3.1+1.20.1
- DrawersTooltip-1.20.1-forge-8.0.0
- easy_villagers-1.20.1-1.0.12
- elytraslot-forge-6.3.0+1.20.1
- entityculling-forge-1.6.2-mc1.20
- FarmersDelight-1.20.1-1.2.3
- Fastload-Reforged-mc1.20.1-3.4.0
- ferritecore-6.0.0-forge
- flatbedrock-forge-3.0.1-build.18+mc1.20.1
- FpsReducer2-forge-1.20-2.5
- gjeb-forge-1.20.1-1.3.0.32
- gpumemleakfix-1.20.1-1.6
- infinitetrading-1.20.1-4.0
- Jade-1.20.1-forge-11.5.1
- JadeAddons-1.20.1-forge-5.2.0
- jei-1.20.1-forge-15.2.0.27
- JustEnoughResources-1.20.1-1.4.0.238
- kotlinforforge-4.4.0-all
- KryptonReforged-0.2.3
- Ksyxis-Forge1.17-1.1
- memoryleakfix-forge-1.17+-1.1.2
- MouseTweaks-forge-mc1.20-2.25
- NaturesCompass-1.20.1-1.11.2-forge
- NoFog-1.3.4+1.20.1-forge
- notenoughcrashes-4.4.6+1.20.1-forge
- okzoomer-forge-1.20-3.0.1
- rubidium-extra-0.5.1+mc1.20.1-build.99
- rubidium-mc1.20.1-0.7.0a
- Saros-Create-Addon-1.20.1-1.7-[Forge]
- Searchables-forge-1.20.1-1.0.1
- sliceanddice-forge-3.1.2
- smoothboot-mc1.20.1-0.0.3
- smoothchunk-1.20.1-3.1
- sophisticatedbackpacks-1.20.1-3.18.59.909
- sophisticatedcore-1.20.1-0.5.89.425
- spark-1.10.53-forge
- starlight-1.1.2+forge.1cda73c
- Steam_Rails-1.5.0+forge-mc1.20.1-build.14
- storagedrawers-1.20.1-12.0.2
- supermartijn642configlib-1.1.8-forge-mc1.20
- supermartijn642corelib-1.1.12a-forge-mc1.20

**Comment by SiverDX** [comment]
Just saw that the 1.20.x branch is doing the same thing I tried for 1.19.2

@way2muchnoise 
For the texture issue see the comment in https://github.com/way2muchnoise/JustEnoughResources/issues/369

**Comment by AlexDerProGamer** [comment]
I patched the bug in my fork:
https://github.com/AlexDerProGamer/JustEnoughResources-VillagerPatched/releases/tag/patched

**Comment by ImpulsiveLad** [comment]
Came here to report this, glad to see an active issue page for once lol

**Comment by YangkeCui** [comment]
> I patched the bug in my fork: https://github.com/AlexDerProGamer/JustEnoughResources-VillagerPatched/releases/tag/patched

Perfect! Thank you.

**Comment by way2muchnoise** [comment]
Fixed in 13b6f55e604fb22d393f3d41b253ba056d7c9ecc

**Comment by MacChuck** [comment]
Any chance you can backport this fix to 1.20.1? With forge & neoforge diverging at 1.20.2, and waiting on other mods, there's potential update issues.

**Comment by way2muchnoise** [comment]
Can you open a new issue for backporting the change? Easier to track than a comment on a fixed issue :)
